### DISCUSSION TITLE
Add "Save and create another" option when creating new fields

### DISCUSSION THREAD

**Discussion by ghost** [craftcms/cms#9726] [Ideas] [answered]
Every time I save a new field, I'm taken back to the fields page. When I'm creating a bunch of fields at a time, which is fairly common, it would be nice to have the option to save and then immediately be able to create another field from the page that loads after I save a field.

This feature already exists for creating new entries.

**Comment by brandonkelly** [comment, marked as answer]
This was already requested by #8671.
